source/php-drivers.txt

```diff
@@ -36,44 +37,32 @@ Welcome to the documentation site for the official MongoDB PHP driver.
 You can add the driver to your application to work with MongoDB in PHP.
 The MongoDB PHP Driver consists of the following components:
 
-- The `extension <https://github.com/mongodb/mongo-php-driver>`__, which
-  provides a low-level API and mainly serves to integrate
-  `libmongoc and libbson <https://www.mongodb.com/docs/languages/c/c-driver/current/>`__ with
-  PHP.
+- `Extension <https://php.net/mongodb>`__:
+  Provides a low-level API and mainly serves to integrate
+  the `C Driver <https://www.mongodb.com/docs/languages/c/c-driver/current/>`__
+  `libmongoc <https://mongoc.org/libmongoc/current/>`__ and
+  `libbson <https://mongoc.org/libbson/current/>`__ libraries with PHP.
```

Comment:
I: [Commas](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#commas)

```suggestion
  the `C Driver <https://www.mongodb.com/docs/languages/c/c-driver/current/>`__,
  `libmongoc <https://mongoc.org/libmongoc/current/>`__, and
  `libbson <https://mongoc.org/libbson/current/>`__ libraries with PHP.
```

Reply:
I meant CDriver to be a adjective for the two libraries. I think this is confusing especially with the links so I'll just remove the C Driver link